Nodejs/Product/TypeScriptSourceMapReader/TypeScriptSourceMapReader.csproj

```diff
@@ -0,0 +1,106 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="dogfood" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ProductVersion>9.0.30729</ProductVersion>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>{61A8D7EF-B83E-4C21-9479-BA832357E42C}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>TypeScriptSourceMapReader</RootNamespace>
+    <AssemblyName>TypeScriptSourceMapReader</AssemblyName>

```

Comment:
Also add this assembly to the installer and to our BuildRelease script for signing